src/rust/ide/engine-api/build.rs

```diff
@@ -0,0 +1,138 @@
+use ensogl_build_utilities::absolute_path;
+
+use std::fs;
+use std::fs::File;
+use std::fs::create_dir_all;
+use std::io::prelude::*;
+use std::path::PathBuf;
+use std::process::Command;
+use zip_extensions::read::ZipArchiveExtensions;
+
+
+
+// =========================
+// == Hardcoded constants ==
+// =========================
+
+
+/// The path where downloaded artefacts will be stored.
+const ZIP_PATH   : &str = "./pkg/fbs-schema.zip";
+/// The path where downloaded artefacts will be unziped.
+const UNZIP_PATH : &str = "./pkg/fbs-upload/fbs-schema/";
+
+/// Commit from `enso` repository that will be used to obtain artefacts from.
+const COMMIT: &str = "25494bbb4315d7d6d625424280dfdee0e49dd045";
+
+/// Obtains a URL where the artefacts can be downloaded.
+pub fn flatbuffers_url() -> reqwest::Url {
+    let url = format!("https://packages.luna-lang.org/fbs-schema/nightly/25494bbb4315d7d6d625424280dfdee0e49dd045/fbs-schema.zip");
+    let err = format!("{} is an invalid URL.",url);
+    reqwest::Url::parse(&url).expect(&err)
+}
```

Comment:
As for docs: I don't know what the "artefacts" are. And `flatbuffers_url` suggests that this is url for downloading the whole library. I would rename it to `interface_description_url`